watchtower-plugin/src/net/http.rs

```diff
@@ -405,9 +524,13 @@ mod tests {
         let (sybil_tower_sk, sibyl_tower_pk) = cryptography::get_random_keypair();
         let appointment = generate_random_appointment(None);
 
-        let appointment_receipt = get_random_appointment_receipt(sybil_tower_sk);
+        #[cfg(feature = "accountable")]
```

Comment:
This whole test should only apply for accountable compiling

Reply:
resolved